lib/ziti-tunnel-cbs/ziti_dns.c

```diff
@@ -701,12 +703,12 @@ ssize_t on_dns_req(const void *ziti_io_ctx, void *write_ctx, const void *q_packe
 
     if (q->type == NS_T_A || q->type == NS_T_AAAA) {
         process_host_req(req);
-    } else {
+    } else if (req->msg.recursive) {
```

Comment:
I worry this may break windows domain stuff. maybe only do this check for upstream?